<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAndReplaceUi</class>
 <widget class="QTabWidget" name="FindAndReplaceUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="basicTab">
   <attribute name="title">
    <string>Sear&amp;ch</string>
   </attribute>
   <layout class="QVBoxLayout" name="dynamicLayoutBasic_">
    <item>
     <layout class="QVBoxLayout" name="findLayout">
      <item>
       <widget class="QLabel" name="findLabel">
        <property name="text">
         <string>&amp;Find:</string>
        </property>
        <property name="buddy">
         <cstring>find_work_area_</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lyx::frontend::EmbeddedWorkArea" name="find_work_area_" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="replaceLayout">
      <item>
       <widget class="QLabel" name="replaceLabel">
        <property name="text">
         <string>Replace &amp;with:</string>
        </property>
        <property name="buddy">
         <cstring>replace_work_area_</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lyx::frontend::EmbeddedWorkArea" name="replace_work_area_" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="caseCB">
        <property name="toolTip">
         <string>Perform a case-sensitive search</string>
        </property>
        <property name="text">
         <string>Case &amp;sensitive</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="findNextPB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>100</horstretch>
          <verstretch>32</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Find next occurrence [Enter]</string>
        </property>
        <property name="text">
         <string>Find &amp;Next</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="wordsCB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Restrict search to whole words only</string>
        </property>
        <property name="text">
         <string>W&amp;hole words</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="replacePB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>100</horstretch>
          <verstretch>32</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Replace and find next occurrence [Enter]</string>
        </property>
        <property name="text">
         <string>&amp;Replace</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="searchbackCB">
        <property name="toolTip">
         <string>Shift+Enter search backwards directly</string>
        </property>
        <property name="text">
         <string>Search &amp;backwards</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="replaceallPB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>100</horstretch>
          <verstretch>32</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Replace all occurrences at once</string>
        </property>
        <property name="text">
         <string>Replace &amp;All</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="advancedTab">
   <attribute name="title">
    <string>S&amp;ettings</string>
   </attribute>
   <layout class="QVBoxLayout" name="dynamicLayoutAdvanced_">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>The scope to which the search horizon is restricted</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Scope</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="CurrentDocument">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>C&amp;urrent document</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="MasterDocument">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Current document and all related documents belonging to the same master document</string>
         </property>
         <property name="text">
          <string>&amp;Master document</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="OpenDocuments">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>All open documents</string>
         </property>
         <property name="text">
          <string>&amp;Open documents</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="AllManualsRB">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>&amp;All manuals</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="1" column="0">
       <widget class="QCheckBox" name="ignoreFormatCB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>If unchecked, the search will be limited to occurrences of the selected text and paragraph style</string>
        </property>
        <property name="text">
         <string>I&amp;gnore format</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="keepCaseCB">
        <property name="toolTip">
         <string>Keep the case of the replacement's first letter as in each matching text first letter</string>
        </property>
        <property name="text">
         <string>&amp;Preserve first case on replace</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="expandMacrosCB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Expand macros</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="OnlyMaths">
        <property name="toolTip">
         <string>Restrict search to math environments only</string>
        </property>
        <property name="text">
         <string>Search on&amp;ly in maths</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lyx::frontend::EmbeddedWorkArea</class>
   <extends>QWidget</extends>
   <header>GuiWorkArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
